tests/baselines/reference/instantiatedTypeAliasDisplay.types

```diff
@@ -0,0 +1,66 @@
+=== tests/cases/compiler/instantiatedTypeAliasDisplay.ts ===
+// Repros from #12066

```

Comment:
Can you please add //@declaration: true to make sure declaration emitter behaves correctly too.
Thanks